flat-manager/index.js

```diff
@@ -9,7 +9,7 @@ class Configuration {
     this.endOfLife = core.getInput('end-of-life')
     this.endOfLifeRebase = core.getInput('end-of-life-rebase')
     // FIXME: get this from the outputs of the flatpak-builder action
-    this.localRepoName = 'repo'
+    this.localRepoPath = 'repo'
```

Comment:
you probably want 
```suggestion
    this.localRepoPath = core.getInput('local-repository') || 'repo'
```

Reply:
I would also check if the path exists and is a directory otherwise error out early